tst/testinstall/opers/FrattiniSubgroup.tst

```diff
@@ -40,8 +40,8 @@ gap> FrattiniSubgroup(CyclicGroup(IsPermGroup, 9));
 Group([ (1,4,7)(2,5,8)(3,6,9) ])
 gap> FrattiniSubgroup(CyclicGroup(IsPcGroup, 3));
 Group([  ])
-gap> FrattiniSubgroup(CyclicGroup(IsPcGroup, 9));
-Group([ f2 ])
+gap> AsSet( FrattiniSubgroup(CyclicGroup(IsPcGroup, 9)) );
```

Comment:
Two different methods are called here depending on which packages (and thus implications are loaded), but both really are fine.